lemur/plugins/lemur_aws/elb.py

```diff
@@ -149,6 +149,38 @@ def get_listener_arn_from_endpoint(endpoint_name, endpoint_port, **kwargs):
         raise
 
 
+@sts_client("elbv2")
+@retry(retry_on_exception=retry_throttled, wait_fixed=2000, stop_max_attempt_number=5)
+def get_load_balancer_arn_from_endpoint(endpoint_name, **kwargs):
+    """
+    Get a load balancer ARN from an endpoint.
+    :param endpoint_name:
+    :return:
+    """
+    try:
+        client = kwargs.pop("client")
+        elbs = client.describe_load_balancers(Names=[endpoint_name])
+        for elb in elbs["LoadBalancers"]:
+            return elb["LoadBalancerArn"]
```

Comment:
why do we iterate over elbs to just return the first `elb["LoadBalancerArn"]`?

Reply:
Changed code